docs-ref-services/latest/servicebus.md

```diff
@@ -15,402 +15,39 @@ ms.service: service-bus
-
-msg1 = Message('Hello World!')
-msg2 = Message('Hello World again!')
-sbs.send_queue_message_batch('taskqueue', [msg1, msg2])
-```
-
-It is then possible to call the **receive\_queue\_message** method to dequeue the message.
-
-```python
-msg = sbs.receive_queue_message('taskqueue')
-```
-
-### Service Bus topics
-
-The **create\_topic** method can be used to create a server-side topic:
-
-```python
-sbs.create_topic('taskdiscussion')
-```
-
-The **send\_topic\_message** method can be used to send a message to a topic:
-
-```python
-from azure.servicebus.control_client import Message
-
-msg = Message(b'Hello World!')
-sbs.send_topic_message('taskdiscussion', msg)
-```
-
-The **send\_topic\_message_batch** method can be used to send  several messages at once:
-
-```python
-from azure.servicebus.control_client import Message
-
-msg1 = Message(b'Hello World!')
-msg2 = Message(b'Hello World again!')
-sbs.send_topic_message_batch('taskdiscussion', [msg1, msg2])
-```
-
-Please consider that in Python 3 a str message will be utf-8 encoded and you should have to manage your encoding yourself in Python 2.
-
-A client can then create a subscription and start consuming messages by calling the **create\_subscription** method followed by the **receive\_subscription\_message** method. Please note that any messages sent before the subscription is created will not be received.
-
-```python
-from azure.servicebus.control_client import Message
-
-sbs.create_subscription('taskdiscussion', 'client1')
-msg = Message('Hello World!')
-sbs.send_topic_message('taskdiscussion', msg)
-msg = sbs.receive_subscription_message('taskdiscussion', 'client1')
-```
-
-### Event Hub
-
-Event Hubs enable the collection of event streams at high throughput, from a diverse set of devices and services.
+## Libraries for data access
 
-The **create\_event\_hub** method can be used to create an event hub:
+The latest version of the Azure Servicebus libraries is version 7.x.x. We highly recommend using version 7.x.x for new applications.
 
-```python
-sbs.create_event_hub('myhub')
-```
+To update existing applications to version 7.x.x, please follow the [migration guide](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/servicebus/azure-servicebus/migration_guide.md).
 
-To send an event:
+### Version 7.x.x
 
-```python
-sbs.send_event('myhub', '{ "DeviceId":"dev-01", "Temperature":"37.0" }')
-```
+The version 7.x.x client library for Python is part of the Azure SDK for Python. The source code for the Azure ServiceBus client libraries for Python is available on [GitHub](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus).
 
-The event content is the event message or JSON-encoded string that contains multiple messages.
+Use the following library to manage your Azure Service Bus resources like namespaces, queues, topics, subscriptions and rules via the Azure Resource Manager, you would use the below package:
```

Comment:
This line is for the "Azure Resource Manager" i.e. the ARM libraries i.e. the mgmt plane one. This should move under the "Libraries for resource management" section